airbyte-api/src/main/openapi/config.yaml

```diff
@@ -2298,6 +2298,15 @@ components:
           $ref: "#/components/schemas/WorkspaceId"
         name:
           type: string
+    SourceDiscoverSchema:
```

Comment:
nit: rename this to something like `SourceDiscoverSchemaRequestBody` to match the convention of other schemas here